ynr/apps/duplicates/templates/duplicates/duplicatesuggestion_list.html

```diff
@@ -4,48 +4,49 @@
 
 {% block content %}
   <h1>Review duplicate suggestions</h1>
-  <table class="table">
-    <thead>
-      <tr>
-        <th>Primary Person (will remain)</th>
-        <th>Person To Be Merged (will be removed)</th>
-        <th>Suggested by</th>
-        <th>Approve</th>
-        <th>Reject</th>
-      </tr>
-    </thead>
-    <tbody>
-      {% for object in object_list %}
 
+  {% for object in object_list %}
+    <p>User <a href="{{ object.user.get_absolute_url }}">{{ object.user }}</a> has suggested that
+      <a href="{{ object.get_absolute_url }}">{{ object.person }}</a> is a duplicate of
+      <a href="{{ object.other_person }}">{{ object.other_person }}</a>.
+    </p>
+    {% include "duplicates/includes/_person_details.html" with person=object.person other_person=object.other_person %}
+    <table class="table">
```

Comment:
Can we have this in the same table, as a new row, label in one cell and buttons in the other two of that row?

Like this:

![Screenshot from 2021-04-26 15-49-52](https://user-images.githubusercontent.com/242329/116103049-22e34100-a6a7-11eb-9f9c-3183e83c1845.png)


(Except there I just have two cells because I'm hacking in the browser).

This will make the actions coupled to the content - I think as different tables it's hard to tell what table the buttons relate to